client/components/tinymce/plugins/contact-form/dialog.jsx

```diff
@@ -1,102 +0,0 @@
-/**
- * External dependencies
- */
-import React, { PropTypes } from 'react';
```

Comment:
Would it be possible to do this as a git move rather than a delete and an add? It would be nice to retain some history here.